tests/kernel/wave/wave_e2e_test.py

```diff
@@ -343,7 +351,7 @@ def repeat(
         # Assert equal does cast to boolean on torch.Tensor
         # which causes issues, hence we cast to numpy before
         # checking.
-        assert_equal(c, ref.values.numpy())
+        assert_allclose(ref, c, atol=0.07)

```

Comment:
That tolerance seems quite high for fp32?

Reply:
lowered to 0.015 :)